src/pe_reports/report_generator.py

```diff
@@ -20,22 +43,74 @@ def load_template():
     return prs
 
 
-def export_set(prs):
+def export_set(prs, out_dir):
     """Export PowerPoint report set to output directory."""
-    pptx_out = "Customer_ID_Posture_Exposure.pptx"
-    prs.save(os.path.join(SAVE_PATH, pptx_out))
+    try:
+        prs.save(os.path.join(out_dir, REPORT_OUT))
+    except FileNotFoundError as not_found:
+        logging.error("%s : Missing input data. No report generated.", not_found)
     return
```

Comment:
This does nothing currently. You should remove any empty `return` statements in the project as they are unnecessary.

```suggestion
```

Reply:
Resolved in 4088918